WordPress/src/main/java/org/wordpress/android/ui/comments/CommentDetailFragment.java

```diff
@@ -992,6 +975,13 @@ public void onFailure(int statusCode) {
                     );
                 }
             });
+
+            // TODO: move following code to subclasses once we extract new fragments
```

Comment:
This should be unblocked by https://github.com/wordpress-mobile/WordPress-Android/pull/20759 :)